revel.go

```diff
@@ -229,6 +229,8 @@ func updateLog(inputmode string) (returnMode string) {
 				packagePathMap[k] = v.(string)
 			}
 		}
+	} else {
+		RevelLog.Warn("Failed to unpack json file", "error", err, "Json", inputmode)
```

Comment:
It should not raise a warning here. Because it said "The input mode can be as simple as "prod" or it can be a JSON string" before. This code should be removed.